net-libs/webkit-gtk/webkit_gtk-2.38.2.recipe

```diff
@@ -231,7 +231,7 @@ BUILD()
 		-DUSE_WPE_RENDERER=OFF \
 		$cmakeDirArgs
 
-	make -C build -j4
+	make -C build -j $jobs
```

Comment:
```suggestion
	make -C build -j $((jobs>4 ? 4 : jobs))
```

Reply:
I have great doubts that our buildmaster will do even with these parameters. Last time I tried to build webkit-gtk, it hung and still does not respond. Maybe it makes sense to build packages on more powerful hardware and put ready-made binary packages in the ports, as it is done for qtwebengine, wine, rust?

Reply:
I don't see how it is related to this patch.

Reply:
then disable the recipe

Reply:
It looks like the HaikuWebKit recipe does not actually use any job-args. Instead it just uses Ninja, which automatically scales jobs based on memory and CPU cores. Those builds take a few hours but it's not too excessive. So, can we just make whatever changes are needed to use Ninja instead of Make, like the HaikuWebKit recipe does?